ubuntu/R/Rprofile.site

```diff
@@ -0,0 +1,12 @@
+# Databricks default configuration for RStudio sessions.
+# Please do not remove this file or modify following lines.
```

Comment:
this is... very opaque to me. Maybe add some more comments to describe what this does?

Reply:
it is very hard to explain. Basically users need these magical commands (all if it is internal API).